app/src/full/java/io/homeassistant/companion/android/notifications/NotificationDeleteReceiver.kt

```diff
@@ -23,5 +41,23 @@ class NotificationDeleteReceiver : BroadcastReceiver() {
         // This maybe the case if timeoutAfter has deleted the notification
         // Then only the empty group is left and needs to be cancelled
         notificationManagerCompat.cancelGroupIfNeeded(group, groupId)
+
+        val data = mutableMapOf(
+            "message" to message
+        )
```

Comment:
We should probably be consistent with what we send into the events.  We normally send the whole data object we receive with the notification not just the message.

Reply:
I had to do some conversion in order to pass `data` properly, but now we get everything :)